regression/symtab2gb/multiple_symtabs/test.desc

```diff
@@ -0,0 +1,10 @@
+CORE
+entry_point.json_symtab
+user.json_symtab library.json_symtab
```

Comment:
Could you please add a readme file or some other explanation describing how the symtab files were generated?